cmd/gnofaucet/gnofaucet.go

```diff
@@ -199,7 +214,16 @@ func serveApp(cmd *command.Command, args []string, iopts interface{}) error {
 		if ip == nil {
 			return
 		}
-		r.ParseForm()
+		r.ParseForm() 
+	
+		// verify catptcha
+		captcha := strings.TrimSpace(r.Form["g-recaptcha-response"][0])
```

Comment:
can you make better handling of errors here, to prevent unwanted panic if someone plays with forms and doesn't send this field?